web/src/lib/types/types.ts

```diff
@@ -76,6 +76,11 @@ export type OnboardingStatus = "SUCCESS" | "ERROR";
 
 export type FormationStepNames = "Name" | "Business" | "Contacts" | "Billing" | "Review";
 export type DbaStepNames = "Business Name" | "DBA Resolution" | "Authorize Business";
+export type GovernmentContractingStepNames =
+  | "NJSTART"
+  | "Apprenticeship Program"
+  | "Public Works"
+  | "Prevailing Wages";
```

Comment:
[sand] Do we need both this and `stepStates()` from `GovernmentContractingPaginator.tsx`?

Reply:
I don't think we need the function.

Reply:
done